language/move-prover/tests/sources/test_vector.move

```diff
@@ -1,4 +1,8 @@
 // dep: tests/sources/stdlib/modules/vector.move
+
```

Comment:
Let us rename this file to simple_vector_client.move.

Reply:
Done.